modules/tinymce/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Icons with a `-rtl` suffix in their name will now automatically be used when the UI is rendered in right-to-left mode #TINY-7782
 
 ### Changed
+- Updated the `cell-border-color` icon #TINY-7697
```

Comment:
I know this is only a draft, but just as a heads up you can skip the changelog for this as it's a new icon that hasn't yet been released 😄

Reply:
Oh, cool, makes sense, thanks! 🙂 I'm fighting the CLA email error so creating these draft PRs. No luck yet. I haven't thought that the team are notified about draft PRs too 😅